content/en/tracing/setup_overview: update OpenTelemetry documentation (#13338)
DataDog/documentation: content/en/tracing/setup_overview/open_standards/_index.md

## OTLP ingest in Datadog Agent

> <div class="alert alert-warning">This functionality is beta and its behavior and configuration may change.</div>

Reviewer: Instead of removing this, let's say that metrics is beta.

```yaml
otlp_config:
  receiver:
    protocols:
      grpc:
      http:
```

The `receiver` section follows the [OpenTelemetry Collector OTLP receiver configuration schema][18].

> You can also enable these via environment variables. For example, the gRPC endpoint could be enabled using the environment variable setting `DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT=0.0.0.0:4317`.

Reviewer 2: ```suggestion
You can also enable these with environment variables. For example, the gRPC endpoint could be enabled using the environment variable setting `DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT=0.0.0.0:4317`.
```

> To view all settings and environment variables supported starting Datadog Agent 7.35.0, please take a look at the `otlp_config` section in [our configuration template][26].

Reviewer 2: ```suggestion
To view all settings and environment variables supported starting Datadog Agent 7.35.0, review the `otlp_config` section in [the configuration template][26].
```

[23]: /tracing/setup_overview/open_standards/python#opentelemetry
[24]: /tracing/setup_overview/open_standards/ruby#opentelemetry
[25]: /tracing/setup_overview/open_standards/nodejs#opentelemetry

> [26]: https://github.com/DataDog/datadog-agent/blob/7.35.0-rc.1/pkg/config/config_template.yaml#L2911-L3054

Reviewer: Comment so that we remember to change to `7.35` once released.

Author: Worst case scenario, even if it stays like this, I don't think the documentation will be wrong between now and then. Or, I could change it now and practically the link will work after we release.
